### ISSUE TITLE
Stray `i` in Croatian .dic file

### ISSUE THREAD

**Issue by avdv** [LibreOffice/dictionaries#4]
Hi.

I was trying to use the hr_HR dictionary with elasticsearch for a hunspell based stemmer.

But, elastic returned an error when creating the index:

```json
{
      "error": {
        "root_cause":[{
          "type":"index_creation_exception",
          "reason":"failed to create index"
        }],
        "type":"unchecked_execution_exception",
        "reason":"java.lang.NumberFormatException: For input string: \"205i\"",
        "caused_by":{
          "type":"number_format_exception",
          "reason":"For input string: \"205i\""
        }
      },
      "status":500
    }
```

In `hr_HR.dic` at line 11960 there is the entry:

```
iglun/205i
```

I suppose it's just a typo as there are no other entries like this?!

**Comment by kaplanlior** [comment]
https://gerrit.libreoffice.org/#/c/43242/

**Comment by fitojb** [comment]
Solved in commit a2a91e4bb5a5b5247afaaecf0c928e9f7025ecbd.

For future reference, please report issues with this repository at [bugs.libreoffice.org](https://bugs.documentfoundation.org) > “LibreOffice” product > “Linguistic” component. This GitHub issue tracker is open by mistake. But of course, thanks for reporting the issue here!

**Comment by avdv** [comment]
Thanks, @fitojb. I am closing this issue.
